guides/plugins/plugins/storefront/using-the-datepicker-plugin.md

```diff
@@ -0,0 +1,190 @@
+# Using the datepicker plugin
+
+## Overview
+
+To provide an input field for date and time values, you can use the datepicker plugin. This guide shows you how to use it.
+
+The datepicker plugin uses the `flatpickr` implementation under the hood. So, check out the `flatpickr` documentation,
+if you need more information about the date picker configuration itself.
+
+<!-- markdown-link-check-disable-next-line -->
+{% embed url="https://flatpickr.js.org" caption="" %}
+
+## Prerequisites
+
+You won't learn how to create a plugin in this guide, head over to our Plugin base guide to create
+your first plugin:
+
+{% page-ref page="../plugin-base-guide.md" %}
+
+You should also know how to customize templates:
+
+{% page-ref page="./customize-templates.md" %}
+
+## Setup a datepicker input field
+
+To apply the datepicker functionality we have to add a DOM element like an input field in a template.
+To keep this example simple for now we just override the `base_main_inner` of the `storefront/page/content/index.html.twig`.
+
+{% code title="<plugin root>/src/Resources/views/storefront/page/content/index.html.twig" %}
+```text
+{% sw_extends '@Storefront/storefront/page/content/index.html.twig' %}
+
+{% block base_main_inner %}
+    <label>
+        <input type="text"
+               name="customDate"
+               class="customDate"
+        />
+    </label>
+
+    {{ parent() }}
+{% endblock %}
+```
+{% endcode %}
+
+Now you should see an empty input field if you open the storefront in your browser.
+We need to add the attribute data-attribute `data-date-picker` to activate the datepicker plugin on our input field.
+
+{% code title="<plugin root>/src/Resources/views/storefront/page/content/index.html.twig" %}
+```text
+{% sw_extends '@Storefront/storefront/page/content/index.html.twig' %}
+
+{% block base_main_inner %}
+    <label>
+        <input type="text"
+               name="customDate"
+               class="customDate"
+               data-date-picker
+        />
+    </label>
+
+    {{ parent() }}
+{% endblock %}
+```
+{% endcode %}
+
+If we check the change in the browser again after reloading the page we now can see, that the datepicker plugin is now active on this element.
+
+## Configure the datepicker
+
+If you select a date with the datepicker from the example above, you will see that a time is always selected and displayed in the input field. By default, the time selection is activated.
+
+We can change this behaviour by passing more options to the datepicker plugin.
+
+Here you can see how this is done by setting up a local Twig variable `pickerOptions`. We can assign a JSON formatted object to the variable and pass the value to the datepicker plugin through the `data-date-picker-options` attribute.
+
+{% code title="<plugin root>/src/Resources/views/storefront/page/content/index.html.twig" %}
+```text
```

Comment:
``````suggestion
```twig
``````

Reply:
Rather HTML I guess? Don't think it can parse twig and will therefore default to `text` anyway.

HTML will at least parse something